deploy-prod/lane.conf

```diff
@@ -0,0 +1,43 @@
+# This is the nginx configuration file located at `/etc/nginx/conf.d/lane.conf`
+
+upstream my_nodejs_upstream {
+        server 127.0.0.1:3000;
+        keepalive 64;
+}
+
+server {
+    listen 80;
+            # listen 443 ssl;
```

Comment:
What does this comment mean?

Reply:
These are for when https is set up, which requires further configuration. Should I delete these or add some additional comment in this file?

Reply:
Added a comment for clarification